ivy-scaling-nginx/docker-compose.yml

```diff
@@ -0,0 +1,52 @@
+version: '3.7'
+services:
+  loadbalancer:
+    image: nginx:stable-alpine
+    ports:
+      - 80:80
+      - 8080:80

```

Comment:
@ivy-arus the port 8080 is not used in testing. The test checks the logs of the ivy container and sees if it finds a line with http://localhost:8080/. It never tries to communicate over port 8080.

Reply:
Ok, removed exposing of port 8080.